site/pages/badger-profile/index.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import React from 'react';
+
+import styles from './style.css';
+import SocialContacts from './social-contacts';
+import Link from '../../components/link';
+
+import type { Badger } from '../../types';
+
+const BlogsLink = ({ badger }: { badger: Badger }) => {
+  if (badger.squarespaceId) {
+    return (<div className={styles.authorLink}>
+      <a href={'/blog/?author=' + badger.squarespaceId}>
+        Read {badger.firstName}&rsquo;s blog posts
+      </a>
+    </div>);
+  }
+  return null;
+};
+
+const BadgerProfile = ({ badger }: { badger: Badger }) => {
+  const fullName = [badger.firstName, badger.lastName].join(' ');
+  badger.categories.sort((a, b) => (a.order - b.order));
+  const categories = badger.categories.map(c => c.name).join(', ');
+  return (
+    <div className={styles.badgerProfile}>
+      <div className={styles.profileContainer}>
+        <div className={styles.profilePictureContainer}>
+          <img
+            src={badger.imageUrl}
+            alt={fullName}
+            className={styles.badgerImage}
+          />
+        </div>
+        <div className={styles.profileDescriptionContainer}>
+          <h1 className={styles.header}>{fullName}</h1>
+          <h2 className={styles.subheader}>{badger.jobTitle}</h2>
+          <h3 className={styles.categories}>{categories}</h3>
+
+          <SocialContacts badger={badger} />
+
+          <div className={styles.about}>
+            {badger.about}
+          </div>
+          <div className={styles.descriptionSections}>
+            {badger.skills.length && [
+              <h2 className={styles.subheader}>Signature Skills</h2>,
+              <p>{badger.skills.join(', ')}.</p>]}
+
+            {badger.achievements && [
+              <h2 className={styles.subheader}>Achievements at Red Badger</h2>,
+              <p>{badger.achievements}</p>]}
+
+            {badger.influence && [
+              <h2 className={styles.subheader}>The thing that changed me</h2>,
+              <p>{badger.influence}</p>]}
+          </div>
```

Comment:
React is still complaining about missing `key` prop (have a look in the console and you'll see the warning). Instead of adding a key prop I think you should get rid of the array syntax here because it's not needed